assets/quill-cursors.scss

```diff
@@ -91,27 +91,20 @@ $transition-func: cubic-bezier(0.250, 0.460, 0.450, 0.940);
     }
   }
 
-  .ql-cursor-flag:hover,
-  .ql-cursor-flag.show-flag,
-  .ql-cursor-caret-container:hover + .ql-cursor-flag {
-    opacity: 1;
-    visibility: visible;
-    transition: none;
-  }
-
   .ql-cursor-flag.no-delay[style] {
     transition-delay: unset !important;
   }
 
   .ql-cursor-caret-container {
-    margin-left: -$spacing-sm;
-    padding: 0 $spacing-sm;
-    z-index: 1;
-
+    z-index: -1;
```

Comment:
I think this might theoretically be breaking. If you have a background on your Quill instance, the carets will no longer be visible.

If we need this, then fine, but we should major version and add this to the changelog.